salt/salt.spec

```diff
@@ -755,6 +755,15 @@ Recommends:     python-pyinotify
 %endif
 %endif
 
+%if 0%{?suse_version}
+%if 0%{?sle_version} >= 150400
+Requires:       %{python_module zypp-plugin if %python-salt}
```

Comment:
We can drop conditional requires here as it became useless.
```
%if 0%{?sle_version} >= 150400
Requires:       %{python_module zypp-plugin if %python-salt}
%else
Requires:       python3-zypp-plugin
%endif
```
can be replaced with
```
%if 0%{?singlespec_compat}
Requires:       %{python_module zypp-plugin}
%else
Requires:       python-zypp-plugin
%endif
```